.changeset/cyan-bears-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@stacks/wallet-web': patch
+---
+
+Fix font feature inconsistency
```

Comment:
We use changesets to provide longer form change log messages. Can you take a look at our [current changelog](https://github.com/blockstack/stacks-wallet-web/blob/main/CHANGELOG.md) and update your message to be more verbose? Ideally we'd avoid shorter commit like messages as we have an ability to give more context/references, etc.

Reply:
Sure I will update that.

Reply:
Let me know if this is fine, it's my first changeset 😄